src/containers/EventListeners/EventListeners.js

```diff
@@ -0,0 +1,220 @@
+import {
+  getEventListeners,
+  getEventListenersErrorMessage,
+  getSelectedNamespace,
+  isFetchingEventListeners,
+  isWebSocketConnected
+} from '../../reducers';
+
+import '../../scss/triggers.scss';
+
+export /* istanbul ignore next */ class EventListeners extends Component {
+  componentDidMount() {
+    this.fetchEventListeners();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { filters, namespace, webSocketConnected } = this.props;
+    const {
+      filters: prevFilters,
+      namespace: prevNamespace,
+      webSocketConnected: prevWebSocketConnected
+    } = prevProps;
+
+    if (
+      !isEqual(filters, prevFilters) ||
+      namespace !== prevNamespace ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchEventListeners();
+    }
+  }
+
+  handleAddFilter = labelFilters => {
+    const queryParams = `?${new URLSearchParams({
+      labelSelector: labelFilters
+    }).toString()}`;
+
+    const currentURL = this.props.match.url;
+    const browserURL = currentURL.concat(queryParams);
+    this.props.history.push(browserURL);
+  };
+
+  handleDeleteFilter = filter => {
+    const currentQueryParams = new URLSearchParams(this.props.location.search);
+    const labelFilters = currentQueryParams.getAll('labelSelector');
+    const labelFiltersArray = labelFilters.toString().split(',');
+    const index = labelFiltersArray.indexOf(filter);
+    labelFiltersArray.splice(index, 1);
+
+    const currentURL = this.props.match.url;
+    if (labelFiltersArray.length === 0) {
+      this.props.history.push(currentURL);
+    } else {
+      const newQueryParams = `?${new URLSearchParams({
+        labelSelector: labelFiltersArray
+      }).toString()}`;
+      const browserURL = currentURL.concat(newQueryParams);
+      this.props.history.push(browserURL);
+    }
+  };
+
+  fetchEventListeners() {
+    const { filters, namespace } = this.props;
+    this.props.fetchEventListeners({
+      filters,
+      namespace
+    });
+  }
+
+  render() {
+    const {
+      error,
+      filters,
+      intl,
+      loading,
+      selectedNamespace,
+      eventListeners
+    } = this.props;
+
+    const initialHeaders = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'date',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.date',
+          defaultMessage: 'Date Modified'
+        })
+      },
+      {
+        key: 'namespace',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.namespace',
+          defaultMessage: 'Namespace'
+        })
+      }
+    ];
+
+    const eventListenersFormatted = eventListeners.map(listener => ({
+      id: `${listener.metadata.namespace}:${listener.metadata.name}`,
+      name: listener.metadata.name,
+      namespace: listener.metadata.namespace,
+      date: (
```

Comment:
The header for this column is 'Date Modified', I suspect we may have this error on some of the other recently added pages too